lib/rspec/mocks/any_instance/stub_chain.rb

```diff
@@ -30,7 +30,7 @@ def invocation_order
             :with => [nil],
             :and_return => [:with, nil],
             :and_raise => [:with, nil],
-            :and_yield => [:with, nil],
+            :and_yield => [:with, :and_yield, nil],
```

Comment:
Would it be safe to say that `with` always comes first, then any combination of the other methods?